index.bs

```diff
@@ -27,7 +27,8 @@ spec: html; urlPrefix: https://html.spec.whatwg.org/multipage/
             text: agent cluster; url: integration-with-the-javascript-agent-cluster-formalism
 spec: storage; urlPrefix: https://storage.spec.whatwg.org/
     type: dfn
-        text: storage shelf; url: storage-shelf
+        text: storage bucket; url: storage-bucket
+        text: storage bottle; url: storage-bottle
```

Comment:
This shouldn't be needed. If you need these exported we should change Storage.

Reply:
Those are not exported indeed. Filed https://github.com/whatwg/storage/issues/153, let's solve it separately.